simulator/src/simulator/simulation-node.ts

```diff
@@ -509,6 +510,58 @@ export class SimulationNode {
     this.onLog.clear()
     this.onError.clear()
   }
+
+  /**
+   * Executes a cli command command via `child_process.exec()` synchronously and returns the stdout and stderr.
+   * If you need the command to be asynchronous, use `executeCliCommandAsync` instead.
+   *
+   * @param args The ironfish cli arguments to execute
+   * @param callback The callback to execute when the command is complete
+   */
+  executeCliCommand(
+    args: string[],
+    callback?: (err: ExecException | null, stdout: string, stderr: string) => void,
+  ): void {
```

Comment:
Why is there no return type? Does the exec return a status code?

Reply:
if the exec fails, status code can be retrieved from `err.status_code`. If it doens't fail, it's guaranteed to be `0`

Reply:
I'll add this into the code comment